### ISSUE TITLE
Enhancment: keep TOC when exporting to HTML

### ISSUE THREAD

**Issue by andrea-vandin** [jupyterlab/jupyterlab-toc#129]
Dear all,

thank you very much for this great plugin. I often export my notebooks in HTML. It would be great to have the TOC preserved in the exported HTML file. At least the numbering of the (sub)-sections

**Comment by vadella** [comment]
If you use [`jupyter_contrib_nbextensions`](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/toc2) you can use `jupyter nbconvert FILE.ipynb --template toc2`

**Comment by stefansimik** [comment]
Yeah, that works, but it is much more complicated - requires installing additional libraries.
On top of that, these libraries are incompatible now and there are a lot of compatibility problems.
It would be much elegant to add TOC on top of the HTML file (or even better in separate left column).
